app/src/main/java/a3/com/convo/activities/LoginActivity.java

```diff
@@ -74,6 +74,7 @@ public void done(List<Page> objects, ParseException e) {
                     Log.e("LoginActivity", "no pages in the server or query failed because objects was empty.");
                     return;
                 }
+                int i = objects.size();
```

Comment:
where is this used?

Reply:
I'm in the process of debugging the duplicate pages error (turns out I just improved it but I didn't completely fix it).